### PR TITLE
fix typos and other errors in EventAggregator

### PR DESCRIPTION
**Pull request by omikron-fp**
Some typo and grammar fixes for `1.x` and `3.x`.

### CODE REVIEW

File: markdown/1.x/en/core-event-aggregator.md [hunk 1]
```diff
@@ -1,12 +1,10 @@
-The `FFCommunicationEventAggregator` (**factfinder.communication.FFCommunicationEventAggregator**) is used to query 
-the FACT-Finder APIs with an event based approach. You can also hook into the communication pipeline.
+The **FFCommunicationEventAggregator** (`factfinder.communication.FFCommunicationEventAggregator`) is used to query the FACT-Finder APIs with an event based approach. You can also hook into the communication pipeline.
```
Comment (torsten-knauf) [inline markdown/1.x/en/core-event-aggregator.md]:
```suggestion
The **FFCommunicationEventAggregator** (`factfinder.communication.FFCommunicationEventAggregator`) is used to query the FACT-Finder API's with an event based approach. You can also hook into the communication pipeline.
```

File: markdown/1.x/en/core-event-aggregator.md [hunk 2]
```diff
@@ -43,11 +41,11 @@ the event. The `url` property is reserved for internal usage.
 ## Hooking into the pipeline
 
 ### `addBeforeDispatchingCallback(fn):string;`
-Adds a callback which executes before every event i.e. search, suggest, recommendation and so on is send.
-The callback receives an event object enriched with basic data i.e. `sid` and if available a
+Adds a callback which executes before every event i.e. `search`, `suggest`, `recommendation` and so on are sent.
```
Comment (torsten-knauf) [inline markdown/1.x/en/core-event-aggregator.md]: Are you sure about that one?

> before every event [...] **is** sent

File: markdown/1.x/en/core-event-aggregator.md [hunk 3]
```diff
@@ -131,9 +129,9 @@ Now you can register/subscribe a callback to the ResultDispatcher with that topi
         firstFetch: <firstFetch:Number>,
         //This defines how many pages per fetch after the first one should be retrieved.(min/default is 1)
         fetchSize: <fetchSize:Number>,
-        //This defines up to witch page the pages should be fetched.(max/default is 10)
+        //This defines up to which page the pages should be fetched.(max/default is 10)
```
Comment (torsten-knauf) [inline markdown/1.x/en/core-event-aggregator.md]:
```suggestion
        // This defines up to which page the pages should be fetched (max/default is 10).
```

File: markdown/1.x/en/core-event-aggregator.md [hunk 4]
```diff
@@ -131,9 +129,9 @@ Now you can register/subscribe a callback to the ResultDispatcher with that topi
         firstFetch: <firstFetch:Number>,
         //This defines how many pages per fetch after the first one should be retrieved.(min/default is 1)
```
Comment (torsten-knauf) [inline markdown/1.x/en/core-event-aggregator.md]:
```suggestion
        // This defines how many pages per fetch after the first one should be retrieved (min/default is 1).
```

File: markdown/1.x/en/core-event-aggregator.md (hunk 1 above)
Comment (omikron-fp) [reply markdown/1.x/en/core-event-aggregator.md]: Without `'` seems to be more common.
https://en.wiktionary.org/wiki/API
https://www.wordhippo.com/what-is/the-plural-of/API.html

File: markdown/1.x/en/core-event-aggregator.md [hunk 5]
```diff
@@ -131,9 +129,9 @@ Now you can register/subscribe a callback to the ResultDispatcher with that topi
...
         fetchSize: <fetchSize:Number>,
-        //This defines up to witch page the pages should be fetched.(max/default is 10)
+        //This defines up to which page the pages should be fetched.(max/default is 10)
         maxFetch: <maxFetch:Number>,
-        //This defines the delay in microseconds between each fetch. (default is 0, this means right after the data is received the next fetch call will be executed)
+        //This defines the delay in milliseconds between each fetch. (default is 0, this means right after the data is received the next fetch call will be executed)
```
Comment (torsten-knauf) [inline markdown/1.x/en/core-event-aggregator.md]:
```suggestion
        // This defines the delay in milliseconds between each fetch. (default is 0, this means right after the data is received the next fetch call will be executed)
```

File: markdown/1.x/en/core-event-aggregator.md (hunk 1 above)
Comment (omikron-fp) [reply markdown/1.x/en/core-event-aggregator.md]:
```suggestion
The **FFCommunicationEventAggregator** (`factfinder.communication.FFCommunicationEventAggregator`) is used to query the FACT-Finder API with an event based approach. You can also hook into the communication pipeline.
```

Comment (torsten-knauf) [reply markdown/1.x/en/core-event-aggregator.md]: I would use the _genitive '_. Whose query are you using?

If meant in plural I would use _APIs'_.

But if you are sure I trust your english ;)

Comment (omikron-fp) [reply markdown/1.x/en/core-event-aggregator.md]: We can even switch to just `API`

Comment (omikron-fp) [reply markdown/1.x/en/core-event-aggregator.md]: By the way, if we were using the genitive here, it would be `FACT-Finder's API`.

Comment (torsten-knauf) [reply markdown/1.x/en/core-event-aggregator.md]: yep, I like the last one most. But never mind for this one - I trust your english ;)

File: markdown/1.x/en/core-event-aggregator.md [hunk 6]
```diff
@@ -125,13 +125,13 @@ Now you can register/subscribe a callback to the ResultDispatcher with that topi
...
     factfinder.communication.FFCommunicationEventAggregator.addFFEvent({
         type: "navigation",
-        //This defines how many pages with the first fetch to the service should be retrieved.(default is 2)
+        // This defines how many pages with the first fetch to the service should be retrieved.(default is 2)
         firstFetch: <firstFetch:Number>,
-        //This defines how many pages per fetch after the first one should be retrieved.(min/default is 1)
+        // This defines how many pages per fetch after the first one should be retrieved.(min/default is 1)
         fetchSize: <fetchSize:Number>,
-        //This defines up to which page the pages should be fetched.(max/default is 10)
+        // This defines up to which page the pages should be fetched.(max/default is 10)
```
Comment (torsten-knauf) [inline markdown/1.x/en/core-event-aggregator.md]: while being at it - should we also so spent an addition space?

Comment (torsten-knauf) [reply markdown/1.x/en/core-event-aggregator.md]:
```suggestion
        // This defines up to which page the pages should be fetched (max/default is 10).
```

**Pull request merged**